.travis.yml

```diff
@@ -6,6 +6,16 @@ matrix:
             python: 3.6
         -   env: TOXENV=py27
             python: 2.7
+        # https://github.com/deadsnakes/travis-ci-python3.7-example
+        -   env: TOXENV=py37
+            sudo: required
+            dist: xenial
+            addons:
+              apt:
+                sources:
+                  - sourceline: 'deb http://ppa.launchpad.net/deadsnakes/ppa/ubuntu xenial main'
```

Comment:
What do you think about https here?

Reply:
it's not necessary for apt, all packages are gpg signed and integrity checked -- for more on this subject: https://unix.stackexchange.com/questions/90227/why-there-is-no-https-transport-for-debian-apt-tool (tl;dr: an attacker can already know what packages you're installing just based on the download size and packages are checked against a signing key)

Also launchpad ppas do not support apt-transport-https

Reply:
Thanks for sharing.